static/app/utils/events.tsx

```diff
@@ -350,6 +351,7 @@ export function getAnalyticsDataForGroup(group?: Group | null): CommonGroupAnaly
     issue_type: group?.issueType ?? IssueType.ERROR,
     issue_status: group?.status,
     issue_age: group?.firstSeen ? getDaysSinceDate(group.firstSeen) : -1,
+    days_since_last_seen: group?.lastSeen ? getDaysSinceDate(group.lastSeen) : -1,
```

Comment:
@malwilley so this isn't going to return a floating point number. I think we should use that here (and for issue_age) given the nature of the importance of granularity timing of issues. Originally, this function was for organization age where we don't need more precision than the day.

Reply:
Makes sense - changed to floats